frontend/packages/dev-console/src/components/topology/__tests__/topology-page.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { StatusBox, LoadingBox } from '@console/internal/components/utils';
+import Topology from '../Topology';
+import { TopologyPage, RenderTopology } from '../TopologyPage';
+import ProjectsExistWrapper from '../../ProjectsExistWrapper';
+import { topologyData } from './topology-test-data';
+
+type TopologyPageProps = React.ComponentProps<typeof TopologyPage>;
+type RenderTopologyProps = React.ComponentProps<typeof RenderTopology>;
+
+let topologyProps: TopologyPageProps;
+let renderTopologyProps: RenderTopologyProps;
+
+describe('Topology page tests', () => {
+  beforeEach(() => {
+    topologyProps = {
+      activeApplication: 'topology-test',
+      knative: false,
+      cheURL: 'che.openshift.io',
+      serviceBinding: false,
+      match: {
+        params: {
+          name: 'topology-test',
+        },
+        isExact: true,
+        path: '',
+        url: '',
+      },
+    };
+
+    renderTopologyProps = {
+      data: topologyData,
+      loaded: true,
+      loadError: '',
+      serviceBinding: false,
+    };
+  });
+
+  it('should render topology page', () => {
+    const wrapper = shallow(<TopologyPage {...topologyProps} />);
+    expect(wrapper.exists()).toBe(true);
+    expect(wrapper.contains(<title>Topology</title>)).toBe(true);
+  });
+
+  it('should render topology list page', () => {
```

Comment:
Looks like this test cases is to verify whether Project list page is displayed when project selected is 'all projects'. But default props passed in this test cases appears to be incorrect. Please verify it.

Reply:
Yes, this is the case where the project list is displayed when all projects option is selected. And props is where ```match.path: ''``` and ```topology-test``` namespace is selected.